base/interactiveutil.jl

```diff
@@ -64,7 +64,7 @@ function edit(path::AbstractString, line::Integer=0)
     nothing
 end
 
-function edit(m::Method)
+function edit(m::TypeMapEntry)
```

Comment:
the conundrum i've had is that users probably mostly care about the mapping `sig+tvars -> method` descriptor that this type implements. while the method itself doesn't generally need the `sig` information and instead wants to know specTypes

fwiw, though, this method can trivially be shortened to take a Method since it only cares about file and line number